CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 Add an entry to the unreleased provider section whenever merging a PR to main that is not targeted at a specific release. These entries will eventually be included in a provider release.
 
+* (refactor) [#1294](https://github.com/cosmos/interchain-security/pull/1294) Remove the equivocation proposal handler.
```

Comment:
Not clear to me when this file gets updated. But seems peculiar we point to this #1294 and not https://github.com/cosmos/interchain-security/pull/1275.